controllers/networking-calico/example/40-managed-resources.yaml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
It is not related but it shows what secrets are created and how the managed resource looks at the end. I think this should be optional documentation for every controller as it aids in understanding how it works, also useful for a quick test.

Reply:
Hm, I disagree. See my other comment: Why do we create multiple secrets at all? Also, this example doesn't contain the real calico resources (like DaemonSet, etc.), so it's not of much help IMHO. Think about the necessity to maintain this. A new developer/unaware person only needs the `Network` CRD to play with this controller. We should rather document in the `README.md` the way it's functioning (i.e., that it will create a `ManagedResource` out of it), but having this example doesn't help much.

Reply:
Again, please remove this example file. It's confusing because nobody has to actively create it. Instead, it will be created by your Calico extension and it will certainly look different. If you want to explain what it does then write a short sentence in the `README.md`.